content/may2015.md

```diff
@@ -36,7 +36,7 @@ Vary: User-Agent
"Condition","interaction":[{"code":"search-type"}],"searchParam":[{"name":"patient","type":"reference","documentation":"Who has the condition?"},{"name":"subject","type":"reference","documentation":"Who has the condition?"},{"name":"clinicalstatus","type":"token","documentation":"The clinical status of the condition"},{"name":"category","type":"token","documentation":"The category of the condition"}]},{"type":"DiagnosticReport","interaction":[{"code":"search-type"}],"searchParam":[{"name":"patient","type":"reference","documentation":"The subject of the report if a patient"},{"name":"subject","type":"reference","documentation":"The subject of the report"}]},{"type":"Encounter","interaction":[{"code":"read"},{"code":"search-type"}],"readHistory":false,"searchParam":[{"name":"patient","type":"reference","documentation":"The patient present at the encounter"}]},{"type":"Immunization","interaction":[{"code":"search-type"}],"searchParam":[{"name":"patient","type":"reference","documentation":"The patient for the vaccination record"}]},{"type":"MedicationPrescription","interaction":[{"code":"search-type"}],"searchParam":[{"name":"patient","type":"reference","documentation":"The identity of a patient to list dispenses for"},{"name":"status","type":"token","documentation":"The status of the prescription"},{"name":"scheduledtiming-bounds-end","type":"date","documentation":"The period end of the prescription timing schedule. This parameter value should be prefixed by \u003c="},{"name":"_count","type":"number","documentation":"The maximum number of results to return"}]},{"type":"Observation","interaction":[{"code":"search-type"}],"searchParam":[{"name":"patient","type":"reference","documentation":"The subject that the observation is about (if patient)"},{"name":"subject","type":"reference","documentation":"The subject that the observation is about"},{"name":"code","type":"token","documentation":"The type of observation"},{"name":"date","type":"date","documentation":"The date range into which the observation falls. This parameter value should be present twice, prefixed once by '\u003e' representing the earliest date and once by '\u003c' representing the latest date."}]},{"type":"Patient","interaction":[{"code":"read"},{"code":"search-type"}],"readHistory":false,"searchParam":[{"name":"_id","type":"token","documentation":"The logical resource id associated with the resource (must be supported by all servers)"},{"name":"birthdate","type":"date","documentation":"The patient's date of birth"},{"name":"identifier","type":"token","documentation":"A patient identifier"},{"name":"name","type":"string","documentation":"A portion of either family or given name of the patient"},{"name":"telecom","type":"string","documentation":"The value in any kind of telecom details of the patient"},{"name":"start","type":"number","documentation":"The offset to use when returning results"},{"name":"_count","type":"number","documentation":"The maximum number of results to return"}]}]}]}
```

Comment:
Besides what's already been stated, in the documentation for Observation's date parameter, '\u003e' and '\u003c' are not rendered as brackets.